data/net/minecraft/Util.mapping

```diff
@@ -30,12 +30,18 @@ CLASS net/minecraft/Util
 	METHOD findPreviousInIterable (Ljava/lang/Iterable;Ljava/lang/Object;)Ljava/lang/Object;
 		ARG 0 iterable
 		ARG 1 current
+	METHOD fixedSize (Ljava/util/List;I)Lcom/mojang/serialization/DataResult;
+		ARG 0 list
+		ARG 1 expectedSize
 	METHOD fixedSize (Ljava/util/stream/IntStream;I)Lcom/mojang/serialization/DataResult;
 		ARG 0 stream
 		ARG 1 size
 	METHOD getPropertyName (Lnet/minecraft/world/level/block/state/properties/Property;Ljava/lang/Object;)Ljava/lang/String;
 		ARG 0 property
 		ARG 1 value
+	METHOD getRandom (Ljava/util/List;Ljava/util/Random;)Ljava/lang/Object;
+		ARG 0 list
```

Comment:
I suggest to be in line with the other `getRandom` methods that this be named `selections` instead.